config/unix-cuda.cmake

```diff
@@ -49,6 +49,7 @@ if( NOT CUDA_FLAGS_INITIALIZED )
     string(APPEND CMAKE_CUDA_FLAGS " --expt-extended-lambda")
   if( CMAKE_CXX_COMPILER_ID MATCHES "XL")
     string(APPEND CMAKE_CUDA_FLAGS " -ccbin xlC -Xcompiler -std=c++14 -Xcompiler --gcc-toolchain=/usr/tce/packages/gcc/gcc-8.3.1 -Xcompiler -qxflag=disable__cplusplusOverride")
+    string(APPEND CMAKE_CUDA_FLAGS " -DCUB_IGNORE_DEPRECATED_CPP_DIALECT -DTHRUST_IGNORE_DEPRECATED_CPP_DIALECT")
```

Comment:
Fixes weird warnings and errors in Thrust (only when XL+nvcc).

Reply:
Thanks for finding these.